packages/pdf-generator/src/utils/styles.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable camelcase */
+import { StyleSheet, Font } from '@react-pdf/renderer';
+import {
+    global_danger_color_100,
+    global_Color_dark_100,
+    chart_global_warning_Color_100,
+    global_disabled_color_100,
+    global_Color_light_300,
+    chart_global_warning_Color_200
+} from '@patternfly/react-tokens';
+import { fontTypes, generateFonts } from './fonts';
+
+Font.register({ family: 'Overpass', fonts: generateFonts(fontTypes) });
+
+export default (style = {}) => StyleSheet.create({
+    ...style,
+    page: {
+        fontWeight: 500,
+        fontFamily: 'Overpass',
+        height: '100%',
+        padding: '20 50'
+    },
+    headerContainer: {
+        display: 'flex',
+        flexDirection: 'row'
+    },
+    currDate: {
+        alignSelf: 'flex-end',
+        marginLeft: 'auto',
+        fontSize: 9,
+        fontStyle: 'italic',
+        color: '#6e6b6c'
+    },
+    reportName: {
+        fontSize: 28,
+        color: global_danger_color_100.value
```

Comment:
it might be okay for now but in the mockup, the colors are different from the PF's react tokens.

Reply:
I've used PF's token colors so we are aligned with the rest of platform.

Reply:
Instead of the global danger color, can we use the red defined color? -- keeps us insulated from if danger color gets modified.
pf-color-red-100